crates/scheduler/src/call/store.rs

```diff
@@ -105,15 +105,15 @@ fn keys<B: Board>(mut call: SchedulerCall<B, api::keys::Sig>) {
             keys
         };
         match keys {
-            Ok(keys) if keys.is_empty() => Ok(0),
+            Ok(keys) if keys.is_empty() => 0,
             Ok(keys) => {
                 let len = keys.len() as u32;
                 let ptr = memory.alloc(2 * len, 2)?;
                 memory.get_mut(ptr, 2 * len)?.copy_from_slice(bytemuck::cast_slice(&keys));
                 memory.get_mut(*ptr_ptr, 4)?.copy_from_slice(&ptr.to_le_bytes());
-                Ok(len)
+                len
             }
-            Err(e) => Err(convert(e)),
+            Err(e) => Err(convert(e))?,
```

Comment:
This could also be removed, in particular, the first try block to build keys could be removed, by using `map_err(convert)?` instead of `?`